app/views/users/index.html.erb

```diff
@@ -0,0 +1 @@
+<h1>User page</h1>
```

Comment:
- [ ] As I mentioned in the previous comment and for the project requirements, kindly make sure for each of the controller actions you have corresponding view files. You are still missing the `app/views/users/show.html.erb` to have all the required files i.e.:
```ruby
    # app/views/posts/index.html.erb
    # app/views/posts/show.html.erb
```
Right now if you test the `user_posts_path   "posts#show"` you will get the below error:
![Screenshot from 2022-11-08 03-15-03](https://user-images.githubusercontent.com/83042742/200450225-f89eb8a8-c5a0-499a-b90c-d6765f746ad7.png)